labelbox/schema/catalog.py

```diff
@@ -23,7 +23,7 @@ def export_v2(
         task_name: Optional[str] = None,
         filters: Union[CatalogExportFilters, Dict[str, List[str]], None] = None,
         params: Optional[CatalogExportParams] = None,
-    ) -> Task:
+    ) -> Union[Task, ExportTask]:
```

Comment:
In this PR we return a union or a tuple in several places, is it possible to avoid that ? Here for instance I can't tell the difference between `Task` and `ExportTask`. Also we end up not using one of the two return values in other places.